ocfweb/docs/docs/services/hpc.md

```diff
@@ -52,22 +52,12 @@ generously funded by a series of grants from the [Student Tech Fund][stf].
 
 ## Slurm
 
-We currently use [Slurm][slurm] as our workload manager for the cluster. We
-will soon post technical documentation about our Slurm configuration, but briefly,
+We currently use [Slurm][slurm] as our workload manager for the cluster. Briefly,
 Slurm is a free and open-source job scheduler which helps distribute jobs from
```

Comment:
Does it help to distribute jobs or do the distributing itself? A bit confusing

How about: "Slurm is a free and open source job scheduler that evenly distributes jobs across an HPC cluster, where each computer in the cluster is referred to as a node."